components/TextFields/tests/snapshot/MDCBaseTextFieldSnapshotTests.m

```diff
@@ -220,4 +220,18 @@ - (void)testEditingTextFieldWithVisiblePlaceholder {
   [self validateTextField:textField];
 }
 
+- (void)testTextFieldWithAssistiveLabelText {
```

Comment:
Since we are adding color: forState: APIs in this PR I was expecting additional snapshots showing what it looks like when we set these values to colors to ensure they continue to affect the visual appearance of our text control. One per state to show off that it works.

Reply:
I updated the existing assistive label snapshot test to have different colors, but I kind of think having a snapshot test for each state is overkill... I did update the unit test to test the leading/trailing accessors on a per state basis though.

Reply:
We are already reducing our code coverage with the hidden features (MDCTextControlAssistiveLabelDrawPriority) in this PR. Lets cover as much as possible in our publicly available APIs so our numbers stay up.

Reply:
The correct coloring of subviews on a per state basis is dependent on three methods: `-setTextControlColorViewModel:forState:`, `-textControlColorViewModelForState:`, and `-applyColorViewModel:withLabelState:`. Maybe if I throughly unit test all three that will be sufficient. I can also add snapshot tests for all three states. I'm happy to do either in the next PR.

At some point, when more of the textfields have landed, I do want to go through and rethink a lot of the snapshot tests I've already added. They're going to change a lot.

Reply:
Ah, it seems like you want more added in this PR rather than in the next one, I can do that 👍